README.md

```diff
@@ -121,9 +128,9 @@ git merge master
 
 # update sentry_sdk's and other modules
 cd react && rm -rf node_modules && npm install
-cd flask && source env/bin/activate && pip install -r requirements.txt
+cd flask && source env/bin/activate && pip3 install -r requirements.txt
 
-# Check that your react/.env, flask/.env and deploy.sh still have correct values
+# Check that your react/local.env, flask/local.env and deploy.sh still have correct values
```

Comment:
Let's remove this line entirely. Back in the day we would change values in `react/.env`, now this is no longer necessary

Reply:
Done!